### ISSUE TITLE
Release of urllib3 version 2 broke anaconda-client

### ISSUE THREAD

**Issue by charris** [numpy/numpy#24042]
See https://github.com/Anaconda-Platform/anaconda-client/issues/654. The temporary fix seems to be something like
```
          # release of urllib3 >=2 broke anaconda-client
          conda install -y anaconda-client 'urllib3<2.0.0'
```
I'm currently testing this with the 1.24.4 release.

**Comment by charris** [comment]
The nightlies uploaded four days ago, so this might have been fixed upstream, but the anaconda-client issue has not been closed. It may have been fixed in an anaconda-client dependency.

**Comment by charris** [comment]
There are other solutions using different channels  where the pin has already been made: https://github.com/dask/dask/pull/10316.

**Comment by charris** [comment]
I fixed this for 1.24.4 by installing an earlier version of urllib3 before building the pinned anaconda-client.
